src/test/java/com/google/roomies/servlets/ListingsDatabaseTest.java

```diff
@@ -101,6 +101,17 @@ public void testAddListingAsMap_listingHasInvalidInput_exceptionThrownBeforeList
     db.addListingAsMap(LISTING_COLLECTION_NAME, listing);
   }
 
+  @Test
```

Comment:
Why is this test here? Was this accidentally omitted from an earlier pull request? There don't seem to be any changes to the class under test in this PR.